templates/etc/nginx/sites-available/fpm.conf.erb

```diff
@@ -41,7 +41,8 @@ server {
 
   location ~ ^(.+\.php)(.*)$ {
     include fastcgi_params;
-
+    #allows to handle 404 in case of a file not found in fpm
+    try_files $uri =404;
```

Comment:
I think it is best to not do it by default. Most php applications don't have it implemented.